### PR TITLE
chore: update to absolute paths when converting README files

### PR DESCRIPTION
**Pull request by janechu**
# Description

This changes relative paths such as `"../../../my-spec-file.md"` to an absolute path to the github repository markdown.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-component-explorer/build/convert-readme.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ const fs = require("fs");
 const MarkdownIt = require("markdown-it");
 
 const srcDir = "../../packages/web-components/fast-components/src/**/README.md";
+// TODO: update this when the repository is renamed
```
[the remark this answers is not in the document]
Comment (janechu) [reply sites/fast-component-explorer/build/convert-readme.js:11]: Unfortunately I don't know any of the specifics to create an issue behind this, only that it is something being considered and the details are not yet worked out. I figured a TODO is better than nothing at this stage.

Comment (awentzel) [inline sites/fast-component-explorer/build/convert-readme.js:11]: We should add issue number with this todo and perhaps include this to the bigger list => https://github.com/microsoft/fast-dna/issues/3307

Comment (janechu) [reply sites/fast-component-explorer/build/convert-readme.js:11]: You left a similar comment which I addressed here: https://github.com/microsoft/fast-dna/pull/3325#discussion_r441218858

File: sites/fast-component-explorer/build/convert-readme.js [hunk 2]
```diff
@@ -33,6 +36,18 @@ const md = new MarkdownIt({
...
+ * Function to determine the correct href path if
+ * it uses a relative path
+ */
+function normalizeLink() {
+    return `href="${webComponentWorkspacePath}`;
+}
+
+function normalizeMarkdown(markdown) {
+    return markdown.replace(/href="\.\.\/\.\.\/\.\.\//gm, normalizeLink);
+}
```
Comment (chrisdholt) [inline sites/fast-component-explorer/build/convert-readme.js:49]: It appears that the trailing `"` is missing in both of these.

Comment (janechu) [reply sites/fast-component-explorer/build/convert-readme.js:49]: That's on purpose, I'm not capturing the full href just the first relative part

Comment (chrisdholt) [reply sites/fast-component-explorer/build/convert-readme.js:49]: party, good to go 👍

**Pull request merged**